src/paths/componentTypes/fetchByName.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Fetch component type by name
+tags:
+  - component-types
+operationId: componentTypesFetchByName
+parameters:
+  - in: path
+    name: name
+    required: true
+    schema:
+      type: string
+responses:
+  '200':
+    description: Returns a component type regarding to the name passed in the path
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/componentTypes/ComponentTypeDto.yaml"
+  "401":
+    $ref: "../../responses/TokenResponse.yaml"
+  "403":
+    $ref: "../../responses/components/ResponseComponentForbidden.yaml"
```

Comment:
normally these calls are not authorized calls, so they can't return 401/403?

Reply:
yes, 
- when there is no token  : The api returns 403 Forbidden.
- When there is wron token ( e.g. format ) : The api returns 401

Reply:
okay, I see. according to our implementation at io.metadew.iesi.server.rest.configuration.security.jwt.JwtWebSecurityConfiguration#configure(org.springframework.security.config.annotation.web.builders.HttpSecurity) the user should be authenticated before calling any endpoint (including action-types, component-types and connection-types)